src/pages/entry.styles.js

```diff
@@ -0,0 +1,23 @@
+import { css } from '@emotion/core'
+import { Color, Space } from 'lib/theme'
+
+const styles = {
+  anchor: css({
+    fontWeight: 600,
+  }),
+  box: css({
+    paddingTop: 0,
```

Comment:
Can you also override `height: 100%` here with `height: auto`? The default 100% is making the page a bit too tall.

Reply:
Good catch, thanks @joshrounsville!